pgo/cmd/create.go

```diff
@@ -240,7 +242,15 @@ func init() {
 	createClusterCmd.Flags().StringVarP(&Password, "password", "w", "", "The password to use for initial database users.")
 	createClusterCmd.Flags().StringVarP(&SecretFrom, "secret-from", "s", "", "The cluster name to use when restoring secrets.")
 	createClusterCmd.Flags().StringVarP(&UserLabels, "labels", "l", "", "The labels to apply to this cluster.")
-	createClusterCmd.Flags().StringVarP(&TablespaceMounts, "tablespaces", "", "", "A list of extra mounts to create for tablespaces. The format is a comma seperated list of <mountName>=<storageConfig>")
+	createClusterCmd.Flags().StringSliceVar(&Tablespaces, "tablespace", []string{}, `Create a PostgreSQL tablespace on the cluster, e.g. "name=ts1:storageconfig=nfsstorage". The format is a key/value map that is delimited by "=" and separated by ":". The following parameters are available:
+
+- name (required): the name of the PostgreSQL tablespace
+- storageconfig (required): the storage configuration to use, as specified in the list available in the "pgo-config" ConfigMap (aka "pgo.yaml")
+- pvcsize: the size of the PVC capacity, which overrides the value set in the specified storageconfig. Follows the Kubernetes quantity format.
+
+For example, to create a tablespace with the NFS storage configuration with a PVC of size 10Gi:
+
+--tablespace=name=ts1:storageconfig=nfsstorage:pvcsize=10Gi`)
```

Comment:
Is it possible to make these a little less 'wide'?

Reply:
@abrightwell I did, but then the formatting looked wacky in the terminal. I could probably just append with `+` and then it would not look wacky.

Reply:
If it's more trouble than it's worth, then I wouldn't worry about it. Just thought I'd ask. :-)

Reply:
@abrightwell It's already done.